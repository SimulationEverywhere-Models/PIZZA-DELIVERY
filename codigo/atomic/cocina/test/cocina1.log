Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para cocina(02)
Mensaje D / 00:00:00:000 / cocina(02) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:01:000 / Root(00) / recibe_pedido /      1.00000 para top(01)
Mensaje X / 00:00:01:000 / top(01) / recibe_pedido /      1.00000 para cocina(02)
Mensaje D / 00:00:01:000 / cocina(02) / 00:00:13:000 para top(01)
Mensaje D / 00:00:01:000 / top(01) / 00:00:13:000 para Root(00)
Mensaje X / 00:00:02:000 / Root(00) / recibe_pedido /      2.00000 para top(01)
Mensaje X / 00:00:02:000 / top(01) / recibe_pedido /      2.00000 para cocina(02)
Mensaje D / 00:00:02:000 / cocina(02) / 00:00:12:000 para top(01)
Mensaje D / 00:00:02:000 / top(01) / 00:00:12:000 para Root(00)
Mensaje X / 00:00:03:000 / Root(00) / recibe_pedido /      3.00000 para top(01)
Mensaje X / 00:00:03:000 / top(01) / recibe_pedido /      3.00000 para cocina(02)
Mensaje D / 00:00:03:000 / cocina(02) / 00:00:11:000 para top(01)
Mensaje D / 00:00:03:000 / top(01) / 00:00:11:000 para Root(00)
Mensaje X / 00:00:04:000 / Root(00) / recibe_pedido /      4.00000 para top(01)
Mensaje X / 00:00:04:000 / top(01) / recibe_pedido /      4.00000 para cocina(02)
Mensaje D / 00:00:04:000 / cocina(02) / 00:00:10:000 para top(01)
Mensaje D / 00:00:04:000 / top(01) / 00:00:10:000 para Root(00)
Mensaje * / 00:00:14:000 / Root(00) para top(01)
Mensaje * / 00:00:14:000 / top(01) para cocina(02)
Mensaje Y / 00:00:14:000 / cocina(02) / sale_pedido /      1.00000 para top(01)
Mensaje D / 00:00:14:000 / cocina(02) / 00:00:03:000 para top(01)
Mensaje Y / 00:00:14:000 / top(01) / sale_pedido /      1.00000 para Root(00)
Mensaje D / 00:00:14:000 / top(01) / 00:00:03:000 para Root(00)
Mensaje * / 00:00:17:000 / Root(00) para top(01)
Mensaje * / 00:00:17:000 / top(01) para cocina(02)
Mensaje Y / 00:00:17:000 / cocina(02) / sale_pedido /      4.00000 para top(01)
Mensaje D / 00:00:17:000 / cocina(02) / 00:00:03:000 para top(01)
Mensaje Y / 00:00:17:000 / top(01) / sale_pedido /      4.00000 para Root(00)
Mensaje D / 00:00:17:000 / top(01) / 00:00:03:000 para Root(00)
Mensaje * / 00:00:20:000 / Root(00) para top(01)
Mensaje * / 00:00:20:000 / top(01) para cocina(02)
Mensaje Y / 00:00:20:000 / cocina(02) / sale_pedido /      3.00000 para top(01)
Mensaje D / 00:00:20:000 / cocina(02) / 00:00:03:000 para top(01)
Mensaje Y / 00:00:20:000 / top(01) / sale_pedido /      3.00000 para Root(00)
Mensaje D / 00:00:20:000 / top(01) / 00:00:03:000 para Root(00)
Mensaje * / 00:00:23:000 / Root(00) para top(01)
Mensaje * / 00:00:23:000 / top(01) para cocina(02)
Mensaje Y / 00:00:23:000 / cocina(02) / sale_pedido /      2.00000 para top(01)
Mensaje D / 00:00:23:000 / cocina(02) / ... para top(01)
Mensaje Y / 00:00:23:000 / top(01) / sale_pedido /      2.00000 para Root(00)
Mensaje D / 00:00:23:000 / top(01) / ... para Root(00)
