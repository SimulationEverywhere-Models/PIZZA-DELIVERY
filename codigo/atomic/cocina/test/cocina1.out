00:00:14:000 sale_pedido 1
00:00:17:000 sale_pedido 4
00:00:20:000 sale_pedido 3
00:00:23:000 sale_pedido 2
